uninstaller/uninstall-helper.ps1

```diff
@@ -55,6 +55,8 @@ $targetFolder = [System.IO.Path]::GetFullPath((Join-Path ($folder) '..'))
 
 echo 'Removing installation folder'
 
+New-Item "$folder\..\temp" -type Directory -Force | Out-Null
+robocopy "$folder\..\temp" "$targetFolder" /purge | Out-Null
 Remove-Item -path "$targetFolder" -Force -Recurse
```

Comment:
This line works for me perfectly on Windows 10.  Making full copy of installation folder (imagine if everything is installed)   before deleting it does not make any sense to me.
What Windows version are you getting error on? What is the error?

Reply:
I am using Windows 7. Powershell version is 5.

error is:

![error](https://user-images.githubusercontent.com/9924513/33411867-f408e6aa-d5ad-11e7-9c79-d6d2a1df4ba4.PNG)

Reply:
Could you test if solution here https://stackoverflow.com/questions/38141528/cannot-remove-item-the-directory-is-not-empty would work?

Reply:
No It's not working it showing error:

![error](https://user-images.githubusercontent.com/9924513/33425436-b1ca1b38-d5e4-11e7-80b0-79babc505d8f.PNG)

Reply:
Got the same on WIndows 10.

Reply:
Fixed in this PR https://github.com/redhat-developer-tooling/developer-platform-install/pull/1084
Could you verify on Windows 7?

Reply:
Still not working on windows 7, it not showing any error but it's not deleting folder.